src/components/ChangePlan/Checkout/Checkout.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import { InjectAppServices } from '../../../services/pure-di';
+import HeaderSection from '../../shared/HeaderSection/HeaderSection';
+import { useIntl } from 'react-intl';
+import { Helmet } from 'react-helmet';
+import { ContactInformation } from './ContactInformation/ContactInformation';
+import { Step } from './Step/Step';
+
+const Checkout = () => {
+  const checkoutSteps = {
+    contactInformation: 'contact-information',
+    billingInformation: 'billing-information',
+    paymentInformation: 'payment-information',
+  };
```

Comment:
This could be moved outside the component right?

Reply:
Done